src/hyperion/experiment_plans/panda_flyscan_xray_centre_plan.py

```diff
@@ -141,7 +141,14 @@ def do_fgs():
             total_exposure,
             30.0,
         )
+        yield from bps.wait()  # Wait for all moves with no explicitly signed group
+        LOGGER.info("kicking off FGS")
         yield from bps.kickoff(fgs_motors)
+        LOGGER.info("Waiting for Zocalo device queue to have been cleared...")
+        yield from bps.wait(
+            ZOCALO_STAGE_GROUP
+        )  # Make sure ZocaloResults queue is clear and ready to accept our new data
+        LOGGER.info("completing FGS")
```

Comment:
Should: I think this is in main now from another issue, can you tidy it up in the merge?